contracts/airdrops/ItemNFTDrop.sol

```diff
@@ -90,11 +87,6 @@ contract ItemNFTDrop is ERC1155, AirdropInfo, AirdropMerkleProof, Ownable {
         return string(abi.encodePacked(baseURI, id));
     }
 
-    //@notice Get the type of airdrop, it's either ERC20, ERC721, ERC1155
-    function getAirdropType() external pure returns (string memory) {
-        return "ERC1155";
-    }
-
     //@notice Checks if the user is eligible for this airdrop
     function isEligibleForReward(uint256 tokenId) external view returns (bool) {
```

Comment:
/// and @return

Reply:
This is also a deprecated contract, keeping it just in case